Osiris/GUI.cpp

```diff
@@ -951,9 +958,18 @@ void GUI::renderMiscWindow() noexcept
         hotkey(config.misc.menuKey);
 
         ImGui::Checkbox("Anti AFK kick", &config.misc.antiAfkKick);
+	ImGui::Checkbox("Block bot", &config.misc.blockbot);
```

Comment:
```suggestion
	ImGui::Text("Blockbot");
      ImGui::SameLine();
      hotkey(config.misc.blockbotkey);           
```
blockbot keybind gui